offchain/hello-world-browser/flake-module.nix

```diff
@@ -181,7 +186,10 @@
         "offchain:hello-world-browser:test" =
           offchain-lib.makeProjectShell hello-world-browser-test { };
       };
-      packages."offchain:hello-world-browser" = hello-world-browser.package;
+      packages = {
+        "offchain:hello-world-browser" = hello-world-browser.package;
+        "offchain:hello-world-browser-ctl-rt-config" = hello-world-browser.packageWithCtlRuntimeConfig { ogmiosConfig = { }; datumCacheConfig = { }; ctlServerConfig = { }; };
```

Comment:
maybe a new entry `offchain:hello-world-browser:serve-ctl-rt-config` in `apps` to host the package?

Reply:
I can add this, but the usefulness is questionable. I added this package for testing, I will remove it in a successive PR. The `packageWithCtlRuntimeConfig` function was introduced such that I can be able to create a nixos-module for  `hello-world-browser`.

Reply:
ahh, I see, I thought this is going to be there running in parallel with `.#offchain:hello-world-browser`